docs/install.rst

```diff
@@ -4,9 +4,61 @@ Install Alpa
 Requirements
 ------------
 - CUDA >= 11.1
-- CuDNN >= 8.1
+- CuDNN >= 8.0.5
 - python >= 3.7
 
+.. _install-from-wheels:
+
+Install from Python Wheels
+--------------------------
+Alpa provides wheels for the following CUDA (CuDNN) and Python versions:
+
+- CUDA (CuDNN): 11.1 (8.0.5), 11.2 (8.1.0), 11.3 (8.2.0)
+- Python: 3.7, 3.8, 3.9
+
+If you need to use other CUDA, CuDNN, or Python versions, please follow the next section to :ref:`install from source<install-from-source>`.
+
+1. To install from wheels, first install Alpa:
+
+  .. code:: bash
+
+    pip3 install --upgrade pip
```

Comment:
Why is this line required?

Reply:
mimicking https://github.com/google/jax#pip-installation-cpu